docs/section23/Rule23-2.md

```diff
@@ -0,0 +1,46 @@
+
+# Airside System - Rule 23-2  
+
+**Schema Version:** 0.0.12  
+**Mandatory Rule:** True  
+**Rule ID:** 23-2  
+**Rule Description:** System 5 & 7 minimum volume setpoint shall be 30% of zone peak airflow, minimum outdoor airflow, or rate required to comply with minium accreditation standards whichever is larger.  
```

Comment:
Consider merging with RDS 23-8 but extending applicability to Systems 6 & 8

Reply:
makes sense.

Reply:
23-2 and 23-8 is merged. 23-8 will be deleted.